src/mudlet.cpp

```diff
@@ -2472,6 +2487,14 @@
     }
 
     return -1;
+#elif defined(Q_OS_UNIX)
+    // Including both GNU/Linux and FreeBSD:
+    // Prefer sysctl() over sysconf() except sysctl() HW_REALMEM and HW_PHYSMEM
+    // return static_cast<int64_t>(sysconf(_SC_PHYS_PAGES)) * static_cast<int64_t>(sysconf(_SC_PAGE_SIZE));
+    struct sysinfo memInfo;
```

Comment:
This is not appropriate IMHO - the ***very next line*** uses a library function (`sysinfo(&memInfo)`) exactly to do the required initialization - and as it is C rather than a C++ (POSIX?) library I do not think it is right to try and do something that is (I *think*) C++ specific.

Reply:
Anyhow - this is exactly the same code as before - just the `#if defined(...)` have been revised to use the Qt `Q_OS_XXX` defines rather than a hodge-podge of more general ones....